website/content/maintainers/community/contributor-growth-framework/motivation.md

```diff
@@ -2,7 +2,6 @@
 title: "Motivating users to contribute"
 linkTitle: "Motivation"
 date: 2020-03-15
-draft: true
 weight: 1
 description: >
    Identify what motivates your users to contribute and how you can encourage more people to contribute.
```

Comment:
I found a couple of small issues. For some reason, GH won't let me review those lines, so I'll list them here :)

* Digital Certificate of Origin should be Developer Certificate of Origin
* The CHAOSS project is a Linux Foundation project, but not a CNCF project.
* CHAOSS (always all caps) isn't actually a tool. It's a set of metrics definitions and group of projects. It might be better to list Augur and GrimoireLab (all one word) as the tools.

Reply:
Would you please submit this as a PR to fix the original document? 

This PR is just to make the page live. If more edits are needed, then let's fix that separately first. Otherwise the person who gets the unlucky task of flipping the draft flag is also then tasked with collecting and implementing all of the last-minute edits. 😀

Reply:
Excellent point :) 

Just to make sure I do this in the right place, I should make PRs against the files in the [community/contributor-growth-framework directory](https://github.com/cncf/tag-contributor-strategy/tree/main/website/content/maintainers/community/contributor-growth-framework)?

I should have some time to do this in my morning (uk time) tomorrow.

Reply:
I made these updates (along with a few others) in PR #116